Libraries/FBReactNativeSpec/FBReactNativeSpec.podspec

```diff
@@ -9,9 +9,11 @@ package = JSON.parse(File.read(File.join(__dir__, "..", "..", "package.json")))
 version = package['version']
 
 source = { :git => 'https://github.com/facebook/react-native.git' }
+scripts_path = "../../node_modules/react-native/scripts"
```

Comment:
usually the files that references `node_modules/react-native` allows overriding it with a `:path` option to work with monorepo projects. not sure if this is required for this file, but it might